docs/migrating.md

```diff
@@ -150,7 +150,7 @@ When converting a Rollout to a Deployment, it involves changing three fields:
 
 When a rollout is referencing to a deployment:
 
-1. Scale-up the Deployment by changing `replicas` field of an existing Rollout to zero.
-1. Scale-down existing Rollout by changing `replicas` field of an existing Rollout to zero.
+1. Scale-up an existing Deployment by changing `replicas` field of an existing Deployment to a desired number of pods.
```

Comment:
Isn't the earlier statement also correct? I think both the things happen. What do you think about this?

```suggestion
1. Scale-up an existing Deployment by changing it's `replicas` field to a desired number of pods and by changing the `replicas` on the Rollout to zero.
```

Reply:
I guess we need to remove the next line (154) if we go ahead with your suggestion, right?

Reply:
@agrawroh WDYT of the above?

Reply:
@gmiroshnykov Sorry for the delay. Yes, let's remove that line and just make it a single sentence. Thanks!

Reply:
@agrawroh I gave it another thought and decided to keep those two lines separate, actually adding a third line to make one thing more explicit: if users will attempt to do this in one go, they might suffer an outage when the Rollouts pods are gone before the Deployment pods are up and running.

But please let me know if I misunderstood it so I can fix it.